src/r5py/util/classpath.py

```diff
@@ -36,7 +36,7 @@
     R5_CLASSPATH = str(pathlib.Path(CACHE_DIR) / pathlib.Path(R5_JAR_URL).name)
     try:
         with open(R5_CLASSPATH, "rb") as jar:
-            assert hashlib.sha256(jar.read()).hexdigest == R5_JAR_SHA256
+            assert hashlib.sha256(jar.read()).hexdigest() == R5_JAR_SHA256
```

Comment:
Shouldn't this be in a separate PR? Did it slip here?

Reply:
Ah, I see you commented about this already

Reply:
Yep. If I could make Git forget all about it I would....